packages/transactions/readme.md

```diff
@@ -0,0 +1,32 @@
+
+# Transactions Manager
+
+Transactions manager for the dock wallet sdk
```

Comment:
Should maybe define what transactions actually are. Probably would require a reference to the Dock token here somewhere.

Reply:
Fixed